python/pyADflow.py

```diff
@@ -1682,6 +1692,146 @@ def Func(Xn, CLstar):
                 break
         return Xn
 
+    def solveTargetFuncs(self, aeroProblem, funcDict, tol=1e-4, nIter=10, Jac0=None):
+        """
+        Solve the an arbitrary set of function-dv sets using a Broyden method.
+
+        Parameters
+        ----------
+        AeroProblem : AeroProblem instance
+            The aerodynamic problem to be solved
+        funcDict : dict
+            Dictionary of function DV pairs to solve...add dv index {'func':{'dv':str, 'dvIdx':idx,'target':val,'initVal':val,'initStep':val}}
```

Comment:
can you explain the keys individually? the dvIdx in particular can be more clear. dv can be the full AP dv name, or just "alpha" etc., and in that case, we append the AP name. Maybe add a TODO comment to add a scaling option for a func as well, since they may not be scaled evenly and we currently check the L2 norm.

Reply:
extended comment